api/envoy/service/ext_proc/v3/external_processor.proto

```diff
@@ -284,11 +302,6 @@ message CommonResponse {
   // HttpHeaders or HttpBody message, but only if this message is returned
   // along with the CONTINUE_AND_REPLACE status.
   config.core.v3.HeaderMap trailers = 4;
-
-  // Clear the route cache for the current request.
-  // This is necessary if the remote server
-  // modified headers that are used to calculate the route.
-  bool clear_route_cache = 5;
```

Comment:
we need to reserve field number.

Reply:
This is a breaking change, like the ext_proc server has to change code to be able to work with Envoy.  With that, do we still need to keep this deprecated field? 
I think the bigger question is that does it worth to make this breaking change if it does not cause functional issue with the fix proposed in https://github.com/envoyproxy/envoy/pull/27657?

Reply:
For 1st question, I think so. Besides, the link I shared above, here also explains [why](https://protobuf.dev/programming-guides/proto3/#deleting) .

For 2nd question, my comment here only apply to this specific proto change. Regarding #27657, IMHO I am ok with proceeding with it  first (as i already approved it :)) while API change is being discussed here , especially if API change will cause wide breakage. But let's see what others think.